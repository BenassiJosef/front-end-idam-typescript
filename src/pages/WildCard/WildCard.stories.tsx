import React from "react";
// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
import { Story, Meta } from "@storybook/react/types-6-0";
import WildCard from "./WildCard";

export default {
  title: "IDAM/Pages/WildCard",
  component: WildCard,
} as Meta;

const Template: Story = () => <WildCard />;

export const WildCardPage = Template.bind({});
