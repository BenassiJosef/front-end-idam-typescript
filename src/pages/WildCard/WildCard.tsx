import React from "react";
import { Header } from "../../components/Header/Header";

const WildCard = (): JSX.Element => {
  return (
    <Header title="Page Not Found" background_color="#fdfdfd" color="#908f8f" />
  );
};
export default WildCard;
