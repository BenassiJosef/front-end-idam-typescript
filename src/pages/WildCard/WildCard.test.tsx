import React from "react";
import WildCard from "./WildCard";
import { render } from "@testing-library/react";

test("wild card component render without crashing", () => {
  const wrapper = render(<WildCard />);
  expect(wrapper).toBeTruthy;
});
