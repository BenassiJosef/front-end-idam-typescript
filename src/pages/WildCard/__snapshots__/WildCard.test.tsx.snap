// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wild card component render without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="sc-bdfBwQ mRGRE"
        color="#908f8f"
      >
        <div
          class="sc-dlfnbm"
        >
          <h1
            class="sc-gsTCUz"
          >
            Page Not Found
          </h1>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="sc-bdfBwQ mRGRE"
      color="#908f8f"
    >
      <div
        class="sc-dlfnbm"
      >
        <h1
          class="sc-gsTCUz"
        >
          Page Not Found
        </h1>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
